client/components/homepage/Features.jsx

```diff
@@ -36,7 +39,8 @@ const Features = () => (
           <div className="col-md-6">
             <div className="icon"><i className="glyphicon glyphicon-comment" /></div>
             <h2>Text Chat</h2>
-            <p>Enter description for this feature</p>
+            <p>Somethings just need to be written down.</p>
```

Comment:
"Some things" over Somethings.
Mention links: "You don't want to spell out that link over voice"